resources/test/fixtures/pyupgrade/UP017.py

```diff
@@ -0,0 +1,11 @@
+import datetime
+import datetime as dt
+from datetime import timezone
+from datetime import timezone as tz
+
+print(datetime.timezone(-1))
+print(timezone.utc)
+print(tz.utc)
+
+print(datetime.timezone.utc)
+print(dt.timezone.utc)
```

Comment:
Just confirming that we don't need to support `from datetime.timezone import utc`...